Cargo.toml

```diff
@@ -16,6 +16,7 @@ std = []
 
 [dependencies]
 bytemuck = { version = "1", optional = true }
+mint = "0.5.5"
```

Comment:
This should also be a dep on "0.5" unless we need something specifically from a point release.

Reply:
Optional feature makes sense with the reduced need.

I think we need at least 0.5.4. I err on the side of specifying at least the current version, since we don't test with older versions. There's no way (afaik) to force Cargo to do minimal version resolution, so it'd be disingenuous to let users install this + mint 0.5.1, for example.

Reply:
Alright, then we can leave this as is.